client/src/components/ReviewDetails.js

```diff
@@ -8,243 +8,249 @@ import {
-          <Box>
-            <form className={classes.ratingForm}>
-              <Box className={classes.ratingTitle}>
-                Tap a star to rate review
-              </Box>
-              <Rating
-                name='rating'
-                value={rating}
-                onChange={(event, newValue) => {
-                  setRating(newValue);
-                }}
-              />
-              <Button
-                type='button'
-                variant='contained'
-                disableElevation
-                color='primary'
-                className={classes.ratingBtn}
-              >
-                Submit
-              </Button>
-            </form>
-          </Box>
+          {selectedReview.comments && (
+            <Box>
+              <form className={classes.ratingForm} onSubmit={handleRating}>
+                <Box className={classes.ratingTitle}>
+                  Tap a star to rate review
+                </Box>
+                <Rating
+                  name='rating'
+                  value={rating}
+                  onChange={(event, newValue) => {
+                    setRating(newValue);
+                  }}
+                />
+                <Button
+                  type='submit'
+                  variant='contained'
+                  disableElevation
+                  color='primary'
+                  className={classes.ratingBtn}
+                >
+                  Submit
+                </Button>
+              </form>
+            </Box>
+          )}
         </Container>
         <Divider />
 
         <Container>
-          <TextareaAutosize
-            rowsMax={10}
-            rowsMin={6}
-            placeholder='Your review'
-            className={classes.code}
-          />
+          <CodeReader code={selectedReview.code} className={classes.code} />
         </Container>
-        <Container>
-          <Box className={classes.avatarHeader}>
-            <Avatar src={img2} className={classes.avatarImg} />
-            <Box>
-              <Typography className={classes.authorName}>
-                Robert Clark
-              </Typography>
-              <Typography className={classes.position}>
-                Senior Developer
+
+        {selectedReview.comments && (
+          <Container>
+            <Box className={classes.avatarHeader}>
+              <Avatar src={img2} className={classes.avatarImg} />
+              <Box>
+                <Typography className={classes.authorName}>
+                  {selectedReview.reviewer.name}
+                </Typography>
+                <Typography className={classes.position}>
+                  Senior Developer
+                </Typography>
+              </Box>
+            </Box>
+            <Box className={classes.reviewContent}>
+              <Typography>
+                It would be great if you add the component like:
               </Typography>
+              <PrismDraft />
             </Box>
```

Comment:
I didn't see the error you told me about. But since you said when you clicked the error was shown, I'm guessing it was because you didn't pass `sendCode` props to `PrismDraft` here.

Reply:
Yes that's true. I mean for the requested review, the displayed code is read-only. But if the user click on it accidentally, it shows error of sendCode. Maybe I will disable it. The CodeReader compoment

Reply:
I'm not sure I understand what you mean. You mean you use `PrismDraft ` to render a read-only text?

Reply:
The CodeReader component. It is for just displaying. But if user accidentally clicks on it. Error of sendCode is shown

Reply:
Oh maybe that's the old `CodeReader`. Because in the updated one (on **_dev_** branch) I removed `sendCode` in `CodeReader `component. But I checked this **_review-routes_** branch, and it's still the older version here.